chapters/ch04.md

```diff
@@ -54,12 +64,13 @@ Here, by “Small NN” we mean a neural network with only a small number of hid
 Thus, you obtain the best performance when you (i) Train a very large neural network, so that you are on the green curve above; (ii) Have a huge amount of data.
 
 ->
-
+Vì thế bạn có thể đạt được kết quả tốt nhất khi (i) huấn luyện mạng neural rất lớn - tương ứng với đường chất lượng màu xanh lục và (ii) huấn luyện với dữ liệu cực lớn. 
 
 
 Many other details such as neural network architecture are also important, and there has been much innovation here. But one of the more reliable ways to improve an algorithm’s performance today is still to (i) train a bigger network and (ii) get more data.
 
 ->
+Còn nhiều chủ đề quan trọng khác như kiến trúc mạng neural. Phần lớn những sáng tạo và công trình nghiên cứu đều tập trung ở đây. Nhưng suy cho cùng, cách đơn giản, hiệu quả và tin cậy để tăng chất lượng thuật toán vẫn là (i) huấn luyện mạng lớn hơn và (ii) tập dữ liệu lớn hơn. 
```

Comment:
- "Phần lớn những sáng tạo và công trình nghiên cứu đều tập trung ở đây" -> ", và đã có rất nhiều sáng tạo trong những chủ đề đó", câu này theo em nghĩ là nó liên quan đến "những chủ đề quan trọng", ý là "có nhiều chủ đề quan trọng ... và đã có rất nhiều sáng tạo trong những chủ đề đó".

- "Nhưng suy cho cùng, cách đơn giản, hiệu quả và tin cậy" -> "Nhưng một trong những cách đáng tin cậy hơn", câu này em nghĩ nên dịch đơn giản sẽ dễ thể hiện cho người đọc..

-  "chất lượng thuật toán" -> "hiệu quả thuật toán" sẽ đúng hơn. Vì theo wiki dịch " Chất lượng là sự phù hợp với nhu cầu", "Hiệu quả là khả năng tạo ra kết quả mong muốn hoặc khả năng sản xuất ra sản lượng mong muốn."

- "(ii) tập dữ liệu lớn hơn. " -> "(ii) thu thập tập dữ liệu lơn hơn", ở đây em nghĩ get more data là một vấn đề đi thu thập dữ liệu.

Reply:
> Còn nhiều chủ đề quan trọng khác, như kiến trúc mạng neural, với rất nhiều sáng tạo và công trình nghiên cứu
> và (ii) lấy thêm dữ liệu.